manifests/ssl.pp

```diff
@@ -2,10 +2,24 @@
 #
 # Apache resources specific for SSL
 #
-class apache::ssl {
+class apache::ssl (
+  $ssl_port            = params_lookup( 'ssl_port' ),
+  $ssl_source          = params_lookup( 'ssl_source' ),
```

Comment:
Honestly I've always used params_lookup on the main class, and not in sub classes, I'm not sure about the params namespace and the functionality when using it in subclasses.
Incidentally I think it's better to have params in the classes where they are used, as here, but that's not a pattern I've used in ng modules.